package wxmsgserver;

message AccessTokenRequest {
    optional string appid  = 1;
    optional string secret = 2;
}

message AccessTokenResponse {
    optional string access_token = 1;
    optional uint32 expired      = 2;
}

service WxMsgServer {
    rpc AccessToken(AccessTokenRequest) returns(AccessTokenResponse);
}

